test/features/resources.feature

```diff
@@ -20,14 +19,12 @@ Feature: Resource UI
         Given "Admin" as the persona
         When I create a resource with name "Non-HTTP link" and URL "git+https://github.com/ckan/ckan.git"
         And I press the element with xpath "//a[contains(@title, 'Non-HTTP link') and contains(string(), 'Non-HTTP link')]"
-        Then I take a screenshot
         And I should see "http://git+https://github.com/ckan/ckan.git"
```

Comment:
Green: should this start with "Then"?

Reply:
As I understand it, the choice of word is purely cosmetic...

Reply:
Yes I think it is, I just saw that the other similar lines in this file had been changed but not this one